shapeshift/web
# feat: fox page trade FOX

## Description

This integrates the trade component into fox page.
Branched from https://github.com/shapeshift/web/pull/1865 which is not merged yet (hence draft PR)

## Notice




## Pull Request Type


## Issue (if applicable)
https://github.com/shapeshift/web/issues/1796


## Risk

No risk


## Testing

- Go to fox page
- Click on Trade tab for FOX
- Notice trade component is displayed as expected
- Click on FOXy tab
- Click on Trade tab for FOXy
- Notice warning text for FOXy trading unavailable and the link to ElasticSwap



## Screenshots (if applicable)

![tradeFox](https://user-images.githubusercontent.com/5720927/170220827-47210172-9f87-407c-b61f-b9d7312f0ef0.png)

![tradeFOXy](https://user-images.githubusercontent.com/5720927/170220852-f3568a1e-95b1-492a-a80d-d57f7bc5fa88.png)
